src/rules/noUnsafeAnyRule.ts

```diff
@@ -416,22 +416,32 @@ class NoUnsafeAnyWalker extends Lint.AbstractWalker<void> {
 }
 
 /** Check if property has no type annotation in this class and the base class */
-function isPropertyAny(node: ts.PropertyDeclaration, checker: ts.TypeChecker) {
-    if (!isNodeAny(node.name, checker) || node.name.kind === ts.SyntaxKind.ComputedPropertyName) {
+function isPropertyAny(
+    node: ts.PropertyDeclaration,
+    checker: ts.TypeChecker,
+    orUnknown: boolean = false,
```

Comment:
`isPropertyAny` is only ever called once. You can remove the `orUnknown` parameter here and replace it with `true`.